koku/masu/external/downloader/azure/azure_report_downloader.py

```diff
@@ -409,6 +412,65 @@ def manifest_date_format(self):
         """Set the Azure manifest date format."""
         return "%Y%m%d"
 
+    def _check_size(self, key, check_inflate=False):
```

Comment:
This method does not apply, even for AWS. Our pods used to be backed by PVCs which had finite sizes. This is not the case any more. We don't have any reason to believe that we cannot uncompress these files.